config/prod/PsychENCODE-Globus-S3.yaml

```diff
@@ -0,0 +1,24 @@
+# Provision a Synapse External Bucket (http://docs.synapse.org/articles/custom_storage_location.html)
+template:
+  type: "http"
+  url: "https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.4.5/templates/S3/synapse-external-bucket.j2"
+stack_name: "PsychENCODE-Globus-S3"
+stack_tags:
+  Department: "SysBio"
+  Project: "PsychENCODE"
+  OwnerEmail: "dan.lu@sagebase.org"
+  CostCenter: "PECDCC-UMass / 106000"
+parameters:
+  # The following parameters are only examples they are not required.
+  # You may omit them if you do not need to override the defaults.
+  # (Optional) true (default) to encrypt bucket, false for no encryption
+  # (Optional) true for read-write bucket, false (default) for read-only bucket
+  AllowWriteBucket: 'true'
+  # (Optional) Synapse username (default: ""), required if AllowWriteBucket=true
+  sceptre_user_data:
+  SynapseIDs:
+    - "3434599"
```

Comment:
`sceptre_user_data:` needs to be at the same level as `parameters:`.  for example:

```
parameters:
  FirstParam: "foo"
  SecondParam: "bar"
sceptre_user_data:
  SynapseIDs:
    - "3434599"
```